queries/snowflake_queries/snowflake_multiple_failed_logins_followed_by_success.yml

```diff
@@ -0,0 +1,32 @@
+AnalysisType: scheduled_rule
+Filename: snowflake_multiple_failed_logins_followed_by_success.py
+RuleID: "Snowflake.Multiple.Failed.Logins.Followed.By.Success"
+Description: >
+  Detecting brute force activity and reporting when a user has incorrectly logged in multiple times and then had a successful login.
+DisplayName: "Snowflake Multiple Failed Logins Followed By Success"
+Enabled: false
+Runbook: Determine if this was a simple mistake or if this is an active brute force attempt.
+ScheduledQueries:
+  - Query.Snowflake.Multiple.Logins.Followed.By.Success
+Severity: Medium
+Tests:
+  -
+    Name: Multiple Failed Logins Followed By Success
+    ExpectedResult: true
+    Log:
+      {
+      "client_ip": "74.00.205.199",
+      "error_codes": [
+        390100
+      ],
+      "error_messages": [
+        "INCORRECT_USERNAME_PASSWORD"
+      ],
+      "match_number": 1,
+      "num_fails": 2,
+      "num_successes": 1,
+      "rows_in_sequence": 3,
```

Comment:
```suggestion
      "num_fails": 6,
      "num_successes": 1,
      "rows_in_sequence": 7,
```

Reply:
I recommend providing test data that would align with the `HAVING` clause in the query statement